jest.config.js

```diff
@@ -6,7 +6,12 @@ module.exports = {
   collectCoverage: true,
   collectCoverageFrom: [
     'packages/**/src/**/*.js',
-    '!packages/terra-functional-testing/src/services/wdio-visual-regression-service/**/*.js',
+    '!packages/terra-functional-testing/src/services/wdio-visual-regression-service/commands/*.js',
+    '!packages/terra-functional-testing/src/services/wdio-visual-regression-service/methods/(S)*.js',
+    '!packages/terra-functional-testing/src/services/wdio-visual-regression-service/modules/(a|b|m)*.js',
+    '!packages/terra-functional-testing/src/services/wdio-visual-regression-service/scripts/*.js',
+    '!packages/terra-functional-testing/src/services/wdio-visual-regression-service/utils/**/*.js',
+    '!packages/terra-functional-testing/src/services/wdio-visual-regression-service/*.js',
```

Comment:
deja vu, hopefully the rebase/merge resolves these

Reply:
it's been merged & cleaned up